pkg/publicapi/client/v2/client.go

```diff
@@ -400,11 +408,6 @@ func (c *httpClient) interceptError(ctx context.Context, err error, resp *http.R
 				WithCode(bacerrors.TimeOutError).
 				WithRetryable()
 		}
-		if netErr.Temporary() {
-			return bacerrors.New("temporary network error").
-				WithCode(bacerrors.NetworkFailure).
-				WithRetryable()
-		}
```

Comment:
why was this removed?

Reply:
@wdbaruni Temporary Error is deprecated and should not be used anymore, especially from a client perespective.

Golang Code:
```
	// Deprecated: Temporary errors are not well-defined.
	// Most "temporary" errors are timeouts, and the few exceptions are surprising.
	// Do not use this method.
	Temporary() bool 
```
Source: https://go-review.googlesource.com/c/go/+/340261/3/src/net/net.go#404